Documentation~/overview.md

```diff
@@ -1,10 +1,10 @@
 # Getting started 
 
-You can access all of ProBuilder's editing tools and actions from the [ProBuilder toolbar](toolbar.md), which dynamically adapts to your [Edit mode](edit-mode-toolbar.md) and your current selection.
+You can access all of ProBuilder's editing tools and actions from:
 
-![Toolbar Example](images/ProBuilderToolbar_GeoActionsArea.png)
-
-Every toolbar button has a tooltip that shows a short summary about its use. Viewing tooltips is a great way to start learning about ProBuilder's functionality.
+* The right-click menu for a ProBuilder Mesh in the Scene view.
```

Comment:
I have to say that when you originally told me "Scene view context menu" I didn't understand what it was. I think it's also not 100% accurate because the mesh has to be selected for those options to turn up, right? If I have a light probe selected, for example, the context menu won't show all the ProBuilder options?

Reply:
That's a good point. If the GameObject doesn't have an attached ProBuilder mesh, you can't access ProBuilder-related options.